### PR TITLE
Patron CLI: Added generation of a .gitignore

### PR DESCRIPTION
**Pull request by azarakovskiy**
## Which problem is this PR solving?

New projects normally require a `.gitignore` file to avoid committing local project files or test output. This would be nice if patron generated it OOB.

## Short description of the changes

- Added a template `.gitignore`
- Copy a template to a generated project

### CODE REVIEW

File: cmd/patron/main.go [hunk 1]
```diff
@@ -184,6 +190,16 @@ func setupGit() error {
 	return exec.Command("git", "init").Run()
 }
 
+func createGitIgnore() error {
+	log.Printf("copying .gitignore")
+	_, err := copyFile("../assets/template.gitignore", ".gitignore")
+
```
Comment (mantzas) [inline cmd/patron/main.go:196]: you can just `return err` here

**Review by mantzas — CHANGES_REQUESTED** [review]

File: cmd/patron/main.go (hunk 1 above)
Comment (azarakovskiy) [reply cmd/patron/main.go:196]: Done

**Comment by azarakovskiy** [comment]
PS: not sure why github closed the initial PR

**Review by mantzas — APPROVED** [review]

**Comment by azarakovskiy** [comment]
@cpapidas @sotirispl

**Review by sotirispl — APPROVED** [review]

**Review by cpapidas — APPROVED** [review]
Good job 👍

**Pull request merged**